An alternative way to load peripherals information (#8)
eclipse-cdt-cloud/vscode-peripheral-inspector: README.md

}
```

> ### Extending SVD Viewer

Reviewer: ```suggestion
### Extending Peripheral Inspector
```

> It is possible to extend the SVD Viewer and register new file extension providers with your VSCode extension. This method will provide reading non-svd formats and load the peripherals information in SVD Viewer. 

Reviewer: ```suggestion
It is possible to extend the Peripheral Inspector and register new file extension providers with your VSCode extension. This method will provide reading non-svd formats and load the peripherals information in Peripheral Inspector. 
```

```json
{
    ...

>     "svdPath": "${workspaceFolder}/STM32F103.customFileExtension"

Reviewer: ```suggestion
    "svdPath": "${workspaceFolder}/STM32F103.<customFileExtension>"
```

Author: Changed.

>     "svdPath": "${workspaceFolder}/STM32F103.<customFileExtension>"

Reviewer: ```suggestion
    "definitionPath": "${workspaceFolder}/STM32F103.<customFileExtension>"
```

    ...
}
```

> For more details about the implementation, please check the [Extending SVD Viewer](./docs/extending-svd-viewer.md) document.

Reviewer: ```suggestion
For more details about the implementation, please check the [Extending Peripheral Inspector](./docs/extending-svd-viewer.md) document.
```
